packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/to/V1_ElementBuilder.ts

```diff
@@ -38,6 +46,25 @@ export type V1_ElementThirdPassBuilder = V1_ElementBuilderPass;
 export type V1_ElementFourthPassBuilder = V1_ElementBuilderPass;
 export type V1_ElementFifthPassBuilder = V1_ElementBuilderPass;
 
+export const V1_checkDuplicatedElement = (
```

Comment:
do we have an existing test for duplication if not lets add.

Reply:
there's one duplication test in `dependency.test`, but I can add one as well in `GraphBuilderFailure.test`